src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/CanonicalPnc.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.services
+
+internal fun String.canonicalPNCNumber(): String {
```

Comment:
I'm interested to know how this works in prison search. So in probation we actually store the offender's PNC twice, so it is *stored* in a canonical form (albeit with both 2 and 4 digit year formats). We do that with a pipeline, not sure if you are doing something similar here, or the NOMIS format is always the same?